server/.eslintrc.js

```diff
@@ -80,6 +74,7 @@ module.exports = {
         "no-bitwise": "error",
         "no-caller": "error",
         "no-cond-assign": "error",
+        "no-console": "error",
```

Comment:
Should we add this rule and the logger to the `/client` directory as well?

Reply:
i can do that for sure, as like a separate logger? if you mean letting the current logger handle the front-end too, i'm not exactly sure how to do it for both frontend and backend since they run separately right now 😔

Reply:
Oh yea like a separate logger but it could just be the same up but duplicated in the /client directory right?

Reply:
oh ok yeah, that makes sense! i'll push the changes later today

Reply:
@aldy97 before i merge this, i just wanna make sure: does front-end need loggers? i added it just in case, but i was wondering if it would actually get used much. its been a while since i worked on front end stuff so i'm not sure 😅

Reply:
@resurreccionl I think it is a bit overhead setting up logger for front-end. But it would a good practice tho.